src/test.rs

```diff
@@ -371,7 +371,7 @@ fn embedded_bytes_receivers() {
     let (super_tx, super_rx) = ipc::channel().unwrap();
     super_tx.send(sub_tx).unwrap();
     let sub_tx = super_rx.recv().unwrap();
-    let bytes = [1, 2, 3, 4, 5, 6, 7, 8];
+    let bytes = [1, 2, 3, 4, 5, 6, 7];
```

Comment:
Add a comment linking to this PR saying that the odd number of bytes is intentional.